### DISCUSSION TITLE
Zed documentation on collaboration features out of date(?)

### DISCUSSION THREAD

**Discussion by SturdyFool10** [zed-industries/zed#24631] [Feature Requests]
### Description

The [Zed documentation on collaboration](https://zed.dev/docs/collaboration) highlights significant differences from the current latest build. Firstly, it doesn't clarify whether initiating a call requires manual steps or if there's a simple invite button for convenience. Secondly, the documentation provides minimal details about the collaboration server, particularly regarding setup and securing it for safe exposure to the wider internet—essential for collaborating with remote users.

### Environment

(the main system is windows, the experience however was the same on Linux, I did check)
Zed: v0.158.0 (Zed Dev cdead5760a2c200179e6a51d5a396c33a3e06e3d)
OS: Windows 10.0.26120
Memory: 31.9 GiB
Architecture: x86_64
GPU: NVIDIA GeForce RTX 3080 || NVIDIA || 566.03

**Comment by SturdyFool10** [comment]
I didn't know how to properly report this, I got flat ignored in the support channel in the discord. this took me a couple days of roaming the user interface to figure out, and it would help other people who are trying to use zed specifically for the collaboration features get going

**Comment by SturdyFool10** [comment]
I should also add: I wanted to get a collab server working but since there is no docs I have been unable to do it, too many questions of: "what does setting X do, how do I make this work with my domain" etc.

**Comment by notpeter** [comment]
This definitely could use some love.  There's been some recent improvements to [docs/src/development/local-collaboration.md](https://github.com/zed-industries/zed/blob/main/docs/src/development/local-collaboration.md?plain=1), but it still needs work.  Happy to review PRs.

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
